pkg/utils/test/webhook.go

```diff
@@ -77,6 +76,7 @@ func ValidationWebhookFailed(causeRegexes ...string) func(*testing.T, *admission
 }
 
 // RunValidationWebhookTests runs a series of ValidationWebhookTestCases
+//
```

Comment:
Do we need this?

Reply:
I'm confused, this line is automatically added by my IDE, I'm trying to understand why 🤔

Reply:
This is actually added by `gofmt` 🤷

Reply:
> This is actually added by gofmt 🤷

Just realised that it is because I'm using [Go 1.19](https://tip.golang.org/doc/go1.19#go-doc):

> Go 1.19 adds support for links, lists, and clearer headings in doc comments. As part of this change, [gofmt](https://tip.golang.org/cmd/gofmt) now reformats doc comments to make their rendered meaning clearer.